index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="assets/index.css" />
+    <link
+      rel="shortcut icon"
+      href="https://upload.wikimedia.org/wikipedia/commons/thumb/f/fa/Apple_logo_black.svg/505px-Apple_logo_black.svg.png"
+      type="image/x-icon"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200&display=swap"
+      rel="stylesheet"
+    />
+    <title>Apple Clone</title>
+  </head>
+  <body>
+    <main class="container">
+      <nav class="navigation">
+        <ul class="nav-list">
+          <li class="link">
+            <a href="#">
+              <img src="assets/img/logo.png" alt="Apple-logo" class="logo"
+            /></a>
+          </li>
+          <li class="link"><a href="#">Store</a></li>
+          <li class="link"><a href="#">Mac</a></li>
+          <li class="link"><a href="#">iPod</a></li>
+          <li class="link"><a href="#">iPhone</a></li>
+          <li class="link"><a href="#">iPad</a></li>
+          <li class="link"><a href="#">iTunes</a></li>
+          <li class="link"><a href="#">Support</a></li>
+        </ul>
+        <form action="search" class="search">
+          <input type="text" placeholder="" class="input" />
+        </form>
+      </nav>
+      <section class="hero">
+        <div class="heading">
+          <h1 class="header1">What will your verse be?</h1>
+          <p>
+            Everyone has something to share. See how people are using iPad in
+            there unique way to add to the world's story
+          </p>
+          <p class="para">
+            <a href="#" class="anchor">
+              Watch the verse film and explore the stories.
+              <img
+                src="./assets/img/play-btn.png"
+                alt="play button"
+                class="play-btn"
+              />
+            </a>
+          </p>
+        </div>
+      </section>
+      <div class="grid">
+        <div class="grid-1"></div>
+        <div class="grid-2"></div>
+        <div class="grid-3"></div>
+        <div class="grid-4"></div>
```

Comment:
- [ ] Please change this div tags for anchor tags (`<a></a>`). In the original webpage, they are links to another part of the page.